sample/ios/Podfile

```diff
@@ -5,14 +5,22 @@ platform :ios, '10.0'
 
 target 'sample' do
   config = use_native_modules!
-  use_react_native!(:path => config["reactNativePath"])
+  use_react_native!(
+    :path => config["reactNativePath"],
+    # to enable hermes on iOS, change `false` to `true` and then install pods
+  	:hermes_enabled => false
```

Comment:
`l`: Why do we need to disable it? Can we maybe add this info as a comment?

Reply:
This is the disabled-by-default from React Native, we should actually have it as true for the sample app.